apps/client/src/common/components/input/popover-picker/PopoverPicker.module.scss

```diff
@@ -14,3 +14,38 @@
   box-shadow: 0 0 0 1px $blue-300;
 }
 
+.icon {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 30px;
+  height: 30px;
+  border-radius: 20px;
+  border: 0.2px solid white;
+  box-shadow: 0 0 0 0.2px $gray-300;
+  cursor: pointer;
+
+  transition-property: box-shadow;
+  transition-duration: $transition-time-action;
+}
+
+.input {
+  color: $gray-200;
+  border: 1px solid transparent;
+  border-radius: 3px;
+  background-color: $gray-1200;
+  padding: 0 0.5rem;
+  font-size: 1rem;
+  height: 2.5rem;
+  outline: none;
+
+  &:hover {
+    background-color: $gray-1100;
+  }
+
+  &:focus {
+    background-color: $gray-1000;
+    color: $gray-50;
+    border: 1px solid $blue-500;
+  }
+}
```

Comment:
great work, I appreciate that you went through the trouble of finding the styles

small correction: the `height` should be `2rem`
we should also remove the top `border-radius` in this context, so `border-radius: 0 0 3px 3px`